ground/openpilotgcs/src/libs/libs.pro

```diff
@@ -13,4 +13,6 @@ SUBDIRS   = \
     qwt \
     sdlgamepad
 
-SUBDIRS +=
+exists( $(OSG_SDK_DIR) ) {
```

Comment:
Yes, you mentioned that before.
What would work with QtCreator?
PS : I use QtCreator only to edit ui files.